src/components/seeds/database/columns.ts

```diff
@@ -319,6 +320,20 @@ export const defaultPreset: Preset = {
   ],
 };
 
+export const defaultImperialPreset: Preset = {
```

Comment:
suggestion, convert `defaultPreset` to a function and pass in boolean imperialSystem.. and on line 318 (imperialSystem ? 'estimatedWeightOunces' : 'estimatedWeightGrams') .. since the only difference between the two is the weight system.. otherwise we have to keep 2 default lists in sync.. just extra maintenance.